### ISSUE TITLE
Character diff color customization

### ISSUE THREAD

**Issue by balwuw** [microsoft/vscode#103207]
VSCode is able to highlight added/removed characters additionally to added/removed lines. The contrast is suboptimal, as the character style is programmatically derived  from the line style. 

It is expected to be able to add extra colors for added/removed characters:
https://stackoverflow.com/questions/49036101/how-to-change-diff-color-visual-studio-code

Actually:
```
	"workbench.colorCustomizations": {
		"[Theme]": {
			"diffEditor.insertedTextBackground": "#00ff0055",
			"diffEditor.removedTextBackground": "#ff000055",
		}
	},
```
This is a design flaw, as these two color tokens are out of synch with the program's capabilities and features. Hence the expected color token scheme is:
```
	"workbench.colorCustomizations": {
		"[Theme]": {
			"diffEditor.insertedLineBackground": "#00ff0055",
			"diffEditor.removedLineBackground": "#ff000055",
			"diffEditor.insertedCharacterBackground": "#88ff8855",
			"diffEditor.removedCharacterBackground": "#ff888855",
		}
	},
```
And the same for `Border`. `Text` should be maintained for backwards compatibility, but it shouldn't stand instead of `Line`, rather as an overall selector.

**Comment by isidorn** [comment]
@hediet maybe maybe we open this up for PRs to add color customisation for diff editors?
fyi @aeschli on color names

**Comment by aeschli** [comment]
Names are good

**Comment by hediet** [comment]
> @hediet maybe maybe we open this up for PRs to add color customisation for diff editors?

Sounds reasonable.

**Comment by sisrfeng** [comment]
There is no such a problem while using  `vim diff `

**Comment by inaryart** [comment]
Related:
https://stackoverflow.com/questions/49036101/how-to-change-diff-color-visual-studio-code
https://github.com/be5invis/vscode-custom-css

we need an official patch
